File: 프로그래머스/unrated/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    answer = ''
    s1 = my_string[num1]
    s2 = my_string[num2]
    for i in range(len(my_string)):
        if i == num1:
            answer += s2
            continue
        if i == num2:
            answer += s1
            continue
        answer+=my_string[i]
    return answer

File: 프로그래머스/Lv.0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
class Solution {
    public String solution(String my_string, int num1, int num2) {
        StringBuffer answer = new StringBuffer(my_string);
        
        char ch1 = answer.charAt(num1), ch2 = answer.charAt(num2);
        answer.delete(num1, num1+1);
        answer.insert(num1, ch2);
        answer.delete(num2, num2+1);
        answer.insert(num2, ch1);
        
        return answer.toString();
    }
}